EverestSplash/EverestSplash.cs

```diff
@@ -376,6 +366,40 @@ private void HandleWindow() {
                     return; // quit asap
                 }
             }
+            
+            // Update state
+            bgBloomPos += 60f*deltaTime; // 60 px per second
+            if (bgBloomPos > realBgH/2f) {
+                bgBloomPos = -realBgH/2f;
+            }
+            
+            wheelAngle += 6.0 * deltaTime; // 6 deg per second
+            // No value reset, it's an angle anyway
+            
+            if (loadingProgress.totalMods != 0) { // if 0 skip updating since it must have not initialized yet
+                float target = (float) loadingProgress.loadedMods / loadingProgress.totalMods * WindowWidth;
+                // ReSharper disable once CompareOfFloatsByEqualityOperator
+                if (prevTarget != target) { // Once the bar has to increase
+                    prevStart = progressWidth; // Start from where it was left off
+                    progressWidthProgress = deltaTime*60; // And go ahead to step 1, there's no point in staying still for a frame
+                    prevTarget = target;
+                }
+                // This is a domain extension from the natural numbers to the reals of
+                // `progressWidth = progressWidth + (target-progressWidth)*0.25f`.
+                // If we relabel it as a function over the naturals: S(n) = S(n-1) + (target-S(n-1))*0.25 and derive
+                // a closed form for S(n), then the trivial domain extension of that function to the reals is used here.
+                // This is due to the need to support different frame rates, since the idea is that an iteration occurs 
+                // each frame at 60fps, so this is the easiest way to approximate it to other (possibly coprime) framerates.
+                progressWidth = target + (prevStart-target) * double.Pow(1-0.25f, progressWidthProgress);
```

Comment:
```suggestion
                progressWidth = target + (prevStart-target) * double.Pow(1-0.25, progressWidthProgress);
```
The division will be done in a 32-bit float because of the float literal, losing precision. Use a double literal instead.